test/common/deployment_types.go

```diff
@@ -183,6 +189,28 @@ func getDeploymentConfiguration(deploymentName string, inst *integreatlyv1alpha1
 		},
 	}
 
+	if inst.Spec.Type == string(integreatlyv1alpha1.InstallationTypeManagedApi) {
+		ratelimitCR := &v12.Deployment{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      quota.RateLimitName,
+				Namespace: Marin3rProductNamespace,
+			},
+		}
+
+		key, err := k8sclient.ObjectKeyFromObject(ratelimitCR)
+		if err != nil {
+			t.Fatalf("Error getting key from ratelimit Deployment: %v", err)
+		}
```

Comment:
If I am looking at the correct source code for `ObjectKeyFromObject()`,
then we don't need any error checking as it never returns errors:
https://github.com/kubernetes-sigs/controller-runtime/blob/b2c90ab82af89fb84120108af14f4ade9df5d787/pkg/client/interfaces.go#L33-L36

Reply:
here is an implementation https://github.com/makslion/integreatly-operator/blob/MGDAPI-1488-A13/vendor/sigs.k8s.io/controller-runtime/pkg/client/interfaces.go#L33-L39

Reply:
Oh, I was looking at newer version of controller-runtime...
Sorry for spam.. 😅